mappings/net/minecraft/client/render/particle/BubbleParticle.mapping

```diff
@@ -0,0 +1 @@
+CLASS cr net/minecraft/client/render/particle/BubbleParticle
```

Comment:
should it be `BubbleParticle` or just `Bubble` 🤔

Reply:
based on the current conventions Bubble may follow them more but idk

Reply:
I'm wary of not using a suffix because there are a few collisions (`PortalParticle` and the tile `Portal`, `TileParticle` and the tile base class)